Enterprise-CMCS/macpro-mdct-mfp
# Setup for DB and API (Merge Me before API Handlers)

### Description


![Screenshot 2023-08-23 at 5 54 46 PM](https://github.com/Enterprise-CMCS/macpro-mdct-mfp/assets/19439679/d8bd737c-5206-4e6d-9bd4-c6b271fc648d)

### Related ticket(s)


MDCT-2718

---
### How to test

`./dev local` and see there are no major errors
`dynamodb-admin listening on http://:::8001 (alternatively http://0.0.0.0:8001)` in another terminal
See that it loads the tables!

### Important updates


2 Dependencies were updated for the upcoming creation of reports in the db.
You'll also need to update your env with whats in the .env-example file.
---
### Author checklist


---
